public/app/features/explore/Logs/Logs.tsx

```diff
@@ -556,7 +579,7 @@ class UnthemedLogs extends PureComponent<Props, State> {
             config.featureToggles.logsExploreTableVisualisation
               ? this.state.visualisationType === 'logs'
                 ? 'Logs'
-                : 'Table'
+                : 'Table (experimental)'
```

Comment:
```suggestion
                : 'Table'
```

I don't think we need this, specially if we feature flag it and do a progressive roll out starting with ops.

Reply:
There was a direct request from Steven to add this text.